hooks/post_gen_project.py

```diff
@@ -56,20 +56,30 @@ def get_random_secret_key():
     chars = "abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)"
     return "".join(secrets.choice(chars) for i in range(50))
 
+web_clients_path = "clients/web"
+mobile_clients_path = "clients/mobile"
 
-def remove_client_files(client):
-    rmtree(join("clients", client))
+def remove_web_client_files(client):
+    rmtree(join(web_clients_path, client))
 
-def move_client_to_root(client):
+def move_web_client_to_root(client):
     if exists("client"):
         # We must be running as an update script
         rmtree("client")
-    move(join("clients", client), join("client"))
-    rmtree(join("clients"))
+    move(join(web_clients_path, client), join("client"))
+    rmtree(join(web_clients_path))
     env_path = join("client", ".env.local.example")
     if exists(env_path):
         rename(env_path, join("client", ".env.local"))
 
+def remove_mobile_client_files(client):
+    rmtree(join(mobile_clients_path,client))
```

Comment:
I guess we don't have a linter setup to run on this file for the PR checks, but this file has a bunch of linting issues.

Reply:
not entirely your fault, so I think you could ignore for now and we could fix that in a lint-only seperate PR

Reply:
Oh maybe my vscode is not configured properly. I didn't get any warnings. I don't usually get into python files

Reply:
I fixed this in a separate PR. You'll likely have a conflict now.

Reply:
There were some other formatting issues in there so I got to use black and then the checks passed!